contracts/RewardDistributor.sol

```diff
@@ -100,6 +100,32 @@ contract RewardDistributor is RewardDistributorStorage, Exponential {
         return msg.sender == admin || msg.sender == address(joetroller);
     }
 
+    /**
+     * @notice Get JOE/AVAX supply speed for a single market
+     * @param rewardType 0 = JOE, 1 = AVAX
+     * @param jToken The market whose reward speed to get
+     * @return The supply reward speed for the market/type
+     */
+    function getSupplyRewardSpeeds(
+        uint8 rewardType,
+        JToken jToken
+    ) public view returns(uint256) {
+       return rewardSupplySpeeds[rewardType][address(jToken)];
+    }
+
+    /**
+     * @notice Get JOE/AVAX borrow speed for a single market
+     * @param rewardType 0 = JOE, 1 = AVAX
+     * @param jToken The market whose reward speed to get
+     * @return The borrow reward speed for the market/type
+     */
+    function getBorrowRewardSpeeds(
+        uint8 rewardType,
+        JToken jToken    
+    ) public view returns(uint256) {
+        return rewardBorrowSpeeds[rewardType][address(jToken)];
+    }
+
```

Comment:
This is not needed:

1. Firstly, it requires us to deploy a new V1 rewarder, which makes no sense.
2. The variables are already public so you don't need separate getter methods.